runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -600,6 +654,117 @@ function LanguageTree:_iter_regions(fn)
   end
 end
 
+---@param region1 Range6[]
+---@param region2 Range6[]
+---@return boolean
+local function region_similar(region1, region2)
```

Comment:
Needs a description. What is "similar"?

Reply:
added some comment 
<https://github.com/neovim/neovim/compare/058d79cf7464732fc4fa5400994c614b49293c70..7b7736ced849114dcab5f3140e1565452d1ddb3d#diff-70b0f3fa7d6baf9e605400a9f0e10431183df821ec4096e8097eaa5f4d5531baR657-R659>
<https://github.com/neovim/neovim/compare/058d79cf7464732fc4fa5400994c614b49293c70..7b7736ced849114dcab5f3140e1565452d1ddb3d#diff-70b0f3fa7d6baf9e605400a9f0e10431183df821ec4096e8097eaa5f4d5531baR795-R806>